src/main/java/fr/sncf/osrd/RollingStock.java

```diff
@@ -15,6 +15,7 @@
     public final double A; // in newtons
     public final double B; // in newtons / (m/s)
     public final double C; // in newtons / (m/s^2)
+    public final Double maxGamma = null;
```

Comment:
Avoid using `Double` because it's an object behind it. Use instead `double` and a special value such as: 

- `Double.NaN`
- `0.`
- `Double.POSITIVE_INFINITY`
- `Double.NEGATIVE_INFINITY`